docs/build-on-polkadot/parachains/asset-hub/register-a-foreign-asset.md

```diff
@@ -0,0 +1,121 @@
+
+## Prerequisites
+
+The Asset Hub parachain is one of the system parachains on a relay chain, such as [Polkadot](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws#/explorer){target=\_blank} or [Kusama](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.api.onfinality.io%2Fpublic-ws#/explorer){target=\_blank}. To interact with these parachains, you can use the [Polkadot.js App](https://polkadot.js.org/apps/#/explorer){target=\_blank} interface. For example:
+
+- [Polkadot Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-polkadot-rpc.dwellir.com#/explorer){target=\_blank}
+- [Kusama Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fsys.ibp.network%2Fstatemine#/explorer){target=\_blank}
+
+For testing purposes, you can also interact with the Asset Hub instance of test networks:
+
+- [Rococo Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-rococo-rpc.dwellir.com#/explorer){target=\_blank}
+- [Paseo Asset Hub](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpas-rpc.stakeworld.io%2Fassethub#/explorer){target=\_blank}
+
+Before you start, ensure that you have: 
+
+- Access to the Polkadot.js App interface, and you are connected to the desired chain
+- A parachain that supports the XCMP protocol to interact with the Asset Hub parachain
+- A funded wallet to pay for the transaction fees and subsequent registration of the foreign asset
+
+This guide will focus on using Polkadot, its local Asset Hub instance, and the [Astar](https://astar.network/){target=\_blank} parachain (`ID` 2006), as stated in the [Test Environment Setup](#test-environment-setup) section. However, the process is the same for other relay chains and their respective Asset Hub parachains, regardless of the network you are using and the parachain owner of the foreign asset.
+
+## Steps to Register a Foreign Asset
+
+1. Open the [Polkadot.js Apps](https://polkadot.js.org/apps/){target=\_blank} interface and connect to the `Asset Hub` parachain
+
+      - For the local environment, connect to Local Node (Chopsticks), check the [Test Environment Setup](#test-environment-setup) section for more details
+      - For the live networks, connect to the `Asset Hub` parachain. You can use either Polkadot, Kusama, Rococo, or Paseo Asset Hub
+
+2. Click on the **Developer** tab on the left sidebar and select the **Extrinsics** section
+
+    ![Access to Developer Extrinsics section](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/register-a-foreign-asset/register-a-foreign-asset-1.webp)
```

Comment:
When using steps in images, they should map to steps in the tutorial. So for example here you don't have something like this:

1. Click on **Developer**
2. Select **Extrinsics**

Therefore, there shouldn't be a 1 and 2 in the image. You can either just point to the Extrinsics item or do something like this:

```
2. Navigate to the **Extrinsics** page
   1. Click on the **Developer** tab from the top navigation bar
   2. Select **Extrinsics** from the dropdown
```

This will turn the steps into a and b though, so ideally you use "a" and "b" in the steps on the screenshot instead of numbers. There were some issues with not being able to center the letters in the step circles, Luca said he was gonna try and play around with it - not sure if he has had time to do so, but maybe you could try it out?